src/renderer/wg_engine/tvgWgRenderer.h

```diff
@@ -59,6 +62,30 @@ class WgRenderer : public RenderMethod
     static WgRenderer* gen();
     static bool init(uint32_t threads);
     static bool term();
+private:
+    // render targets array
+    Array<RenderData> mRenderDatas{};
+private:
+    // surface
+    Surface mTargetSurface = { nullptr, 0, 0, 0, ColorSpace::Unsupported, true };
+    float   mViewMatrix[16]{};
+    // webgpu instances (maybe, create separated entity?)
+    WGPUInstance  mInstance{};
+    WGPUAdapter   mAdapter{};
+    WGPUDevice    mDevice{};
+    WGPUQueue     mQueue{};
+    // webgpu surface (maybe, create separated entity?)
+    WGPUSurface     mSurface{};
+    WGPUSwapChain   mSwapChain{};
+    WGPUTexture     mStencilTex{};
+    WGPUTextureView mStencilTexView{};
+private:
+    // brush pipelines
```

Comment:
thorvg have no "brush" terms in features.
You might wanna use "rendering feature pipelines"